test/integration/enumerate_databases.test.ts

```diff
@@ -0,0 +1,119 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../src';
+
+describe('listDatabases', function () {
+  let client: MongoClient;
+
+  beforeEach(async function () {
+    client = this.configuration.newClient();
+    await client.connect();
+  });
+
+  afterEach(async function () {
+    await client?.close();
+  });
+
+  it('should return an Array', async () => {
+    const dbInfo = await client.db().admin().listDatabases();
+    expect(Array.isArray(dbInfo.databases)).to.be.true;
+  });
+
+  it('should contain no duplicates', async () => {
+    const dbInfo = await client.db().admin().listDatabases();
+    const databaseNames = dbInfo.databases.map(({ name }) => name);
+    const databaseNamesSet = new Set(databaseNames);
+    expect(databaseNames).to.have.lengthOf(databaseNamesSet.size);
+  });
+
+  // TODO(NODE-3860): Create driver test variants that require AUTH enabled
+  describe('authorizedDatabases flag', function () {
+    const username = 'a';
+    const password = 'b';
+    const mockAuthorizedDb = 'mockAuthorizedDb';
+
+    let client: MongoClient;
+    let authorizedClient: MongoClient;
+
+    const authorizedUserOptions: AddUserOptions = {
+      roles: [{ role: 'read', db: mockAuthorizedDb }]
+    };
+
+    beforeEach(function () {
+      if (process.env.AUTH !== 'auth') {
+        this.currentTest.skipReason =
+          'TODO(NODE-3860): Create driver test variants that require AUTH enabled';
+        this.skip();
+      }
+    });
+
+    beforeEach(async function () {
+      // pass credentials from cluster_setup's mlaunch defaults
+      // TODO(NODE-3860): pass credentials instead based on environment variable
+      client = this.configuration.newClient({ auth: { username: 'user', password: 'password' } });
```

Comment:
question @nbbeeken  - based on our conversation this morning, this test will need to be adjusted when I make changes to the cluster-setup script to use bob/pwd123 instead of the default user and password?

Reply:
It should probably just switch to using `newClient()` with no args, and the configuration class will handle using the right connection string or setting the auth settings appropriately.